### ISSUE TITLE
Auto-closing menus

### ISSUE THREAD

**Issue by roelvangils** [godbout/Wooshy.docs#48]
[📹 Screencast](https://scr.11ways.be/Y6HIFx/)

In this screencast, I activated a button that opened a menu / pop-out thing when clicked. The menu closes when I use the trigger shortcut for Wooshy again. Do you think it is possible to keep these things open somehow, so you can also click the things inside with Wooshy?

**Comment by archetyped** [comment]
See Wooshy's [Roadmap](https://github.com/godbout/Wooshy.docs#v1)

**Comment by roelvangils** [comment]
Yessir.

**Comment by godbout** [comment]
yes @archetyped is right: https://github.com/godbout/Wooshy.docs#v1

i'll work on something related but after the v1 release. basically the issue currently is that when The Input shows up, it steals focus so that you can type your search. that makes some types of UI element hide automatically. there's more of those in Ventura also coz it's getting iOS style everywhere, so that will definitely be needed, and implemented. maybe with a system where you can keep typing rather than having to press the keyboard shortcut over and over. not sure. will still have to explore the UX.

**Comment by godbout** [comment]
this issue is the same as: https://github.com/godbout/Wooshy.docs/issues/36

**Comment by godbout** [comment]
working on this, but this is bring other issues (not talking about the huge amount of work to get this to work properly, mostly a complete redo of the event system), like not being able to grab keys on password fields:



still on the stage of playing around and exploring. but hmm the UX kinda makes me wonder if it's worth doing actually. most menus (that disappear) are available or with kV, or by just typing where you wanna go. the "auto closing" stuff i was most interested about was those widget shits related to Control Center. wondering which way the next macOS is gonna go tho... if they add more of those shits then yeah, may be worth adding. not sure. kinda lost here lol

**Comment by godbout** [comment]
also i'm realizing in your case it wouldn't even help coz we can't target menu items lol
